src/plugins/intel_cpu/src/nodes/eltwise.cpp

```diff
@@ -840,8 +848,15 @@ struct jit_uni_eltwise_generic : public jit_uni_eltwise_kernel, public jit_gener
                 uni_vmovups(op, vmm_dst);
                 break;
             case ov::element::bf16:
-                uni_vcvtneps2bf16->emit_code({static_cast<size_t>(vmm_dst.getIdx())}, {static_cast<size_t>(ymm_dst.getIdx())});
-                vmovdqu16(op, ymm_dst);
+                if (isa == x64::avx512_core) {
+                    uni_vcvtneps2bf16->emit_code({static_cast<size_t>(vmm_dst.getIdx())},
+                                                 {static_cast<size_t>(ymm_dst.getIdx())});
+                    vmovdqu16(op, ymm_dst);
+                } else {
+                    uni_vcvtneps2bf16->emit_code({static_cast<size_t>(vmm_dst.getIdx())},
+                                                 {static_cast<size_t>(xmm_dst.getIdx())});
+                    uni_vmovdqu(op, xmm_dst);
+                }
```

Comment:
searching `uni_vcvtneps2bf16->emit_code` in cpu plugin's code, I saw few other places where it's being called, (like normalize.cpp/reduce.cpp/....), why you only extend to avx2 case in here?

Reply:
have created a ticket CVS-127496 to cover the leftover Ops which are not performance critical for current KPI models